response_operations_social_ui/templates/social-change-response-status.html

```diff
@@ -48,7 +48,7 @@ <h1 class="jupiter">Change response status</h1>
                     <button type="submit" id="response-status-change-confirm-button" class="btn btn--primary">Set new status</button>
                   </div>
                   <div>
-                    <a href="" onclick="history.back(); return false;">Cancel</a>
+                    <a href="{{ request.referrer }}">Cancel</a>
```

Comment:
fyi, some security policies may stop this from working, see https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Referrer-Policy. i think it will be ok for this app though as we currently don't set any Referrer-Policy headers